Inconsistent between API guide and message from assert.equal

Could you verify why API guide says something different than message from exception ?

Guide:
```
.equal(actual, expected, [message])
```

Example:
```
// test case
assert.equal(result[0].status, 405, 'Response HTTP status equal 405');
// message when assert.equal fire exception
AssertionError: Response HTTP status equal 405: expected 404 to equal 405
```

@sbidolach Looks like a bug to me.

```js
it("expect equal with comment at start", () => {
  expect(42, "meep").to.equal(43);    // meep
});

it("expect equal with comment at end", () => {
  expect(42).to.equal(43, "meep");    // meep
});

it("assert strictEqual", () => {
  assert.strictEqual(42, 43, "meep"); // meep
});

it("assert equal", () => {
  assert.equal(42, 43, "meep");       // AssertionError: meep: expected 42 to equal 43
});
```

--- a/lib/chai/interface/assert.js
+++ b/lib/chai/interface/assert.js
@@ -129,6 +129,7 @@ module.exports = function (chai, util) {
       , 'expected #{this} to not equal #{act}'
       , exp
       , act
+      , true
     );
   };
 
@@ -156,6 +157,7 @@ module.exports = function (chai, util) {
       , 'expected #{this} to equal #{act}'
       , exp
       , act
+      , true
     );
   };
 

